UI/login.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.6.3/css/all.css" integrity="sha384-UHRtZLI+pbxtHCWp1t77Bi1L4ZtiqrqD80Kn4Z8NTSRyMA2Fd33n5dQ8lWUE00s/"
+        crossorigin="anonymous">
+    <link rel="stylesheet" href="./css/style.css">
+    <link rel="shorcut icon" type="image/x-icon" href="./images/title-icon.png">
+    <title class="fa-fa-pencil">Questioner</title>
+</head>
+
+<body>
+    <nav>
+        <span class="nav left">
+            <a href="index.html" class="fas fa-home"></a>
+        </span>
+        <a href="" class="left">MEETUPS</a>
+        <a href="signup.html" class="right">REGISTER</a>
+    </nav>
+    <div class="form-container">
+        <p style="font-weight:bolder;font-size: 2em;left: 30%;">Log In</p>
+        <form action="">
+            <label for="">Email</label><br>
+            <input type="text"><br>
+            <label for="">Password</label><br>
+            <input type="password"><br><br>
+            <input type="button" class="button" value="Signup">
+        </form>
+    </div>
+    <footer>
+        &copy;Questioner 2019
+    </footer>
+</body>
+
+</html>
```

Comment:
new line

Reply:
Thanks, what is the importance of the new line?